packages/eslint-config/package.json

```diff
@@ -2,19 +2,27 @@
   "name": "@nuxt/eslint-config",
   "version": "0.3.0-beta.5",
   "description": "ESLint config for Nuxt projects",
-  "repository": "nuxt/eslint",
+  "repository": { 
+    "type": "git", 
+    "url": "git+https://github.com/nuxt/eslint.git",
+    "directory": "packages/eslint-config"
+  },
   "license": "MIT",
-  "main": "./dist/legacy.cjs",
   "exports": {
     ".": {
-      "import": "./dist/legacy.mjs",
-      "require": "./dist/legacy.cjs"
-    },
-    "./flat": {
+      "types": "./dist/flat.d.ts",
       "import": "./dist/flat.mjs",
       "require": "./dist/flat.cjs"
+    },
+    "./legacy": {
+      "types": "./dist/legacy.d.ts",
+      "import": "./dist/legacy.mjs",
+      "require": "./dist/legacy.cjs"
```

Comment:
Legacy first is intentionally here to keep backward compatibility (also the legacy format can't load sub modules)

Reply:
Are we planning to eventually change it (e.g. when ESLint v9 gets out of RC)?

Reply:
No, as I said, it's not compatible with the legacy format, so we have to keep the default export to the legacy - at some point, we could emit a warning message and ask to change over, but not right now as the ecosystem is still catching up. With the nuxt module, users don't usually need to care about this anyway